src/Controller/WebController.php

```diff
@@ -104,20 +83,16 @@ public function searchAction(Request $req, SearchClient $algolia, string $algoli
         $typeFilter = str_replace('%type%', '', $req->query->get('type'));
         $tagsFilter = $req->query->get('tags');
 
-        $filteredOrderBys = $this->getFilteredOrderedBys($req);
-
-        $this->computeSearchQuery($req, $filteredOrderBys);
+        $this->computeSearchQuery($req);
 
         if (!$req->query->has('search_query') && !$typeFilter && !$tagsFilter) {
             return (new JsonResponse([
                 'error' => 'Missing search query, example: ?q=example'
             ], 400))->setCallback($req->query->get('callback'));
         }
 
-        $form = $this->createForm(SearchQueryType::class, new SearchQuery());
-
         $index = $algolia->initIndex($algoliaIndexName);
-        $query = '';
+        $query = $req->get('search_query')['query'] ?? '';
```

Comment:
don't use `$request->get()` to read query parameters (this method reads more stuff)